metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -111,6 +112,10 @@ import org.eclipse.{lsp4j => l}
  * @param sh
  *   Scheduled executor service used for scheduling tasks. This class DO NOT
  *   manage the lifecycle of this executor.
+ * @param client
+ *   Metals client used for sending notifications to the client. This class DO
+ *   NOT manage the lifecycle of this client. It is the responsibility of the
+ *   caller to shut down the client.
```

Comment:
Emphasise that MetalsLspService will not call shutdown/cancel on passed language client